week3_cardgame/cardgame/src/components/card/CardStyle.jsx

```diff
@@ -0,0 +1,38 @@
+import { styled } from 'styled-components';
+import pattern from './../../assets/images/pattern.jpg';
+
+export const CardWrapper = styled.section`
+    display: inline-grid;
+
+    // 카드 공통 속성
+    * {
+        grid-area: 1 / 1 / 1 / 1;
+        width: 20rem;
+        height: 25rem;
+        padding: 1rem;
+        backface-visibility: hidden; // 뒤집으면 안보이도록
+        cursor: pointer;
+        border: 10px outset ${(props) => props.theme.colors.pink};
```

Comment:
이 부분도 구조분해 할당을 이용하면 더 간단하게 작성이 가능합니다!!
```suggestion
        border: 10px outset ${({ theme }) => theme.colors.pink};
```

Reply:
오!!!!!!!!!!!!!!!!!!!!!!!!!!!!!감사합니다!
![FreakOutOhMyGodGIF](https://github.com/NOW-SOPT-WEB/ChaeHyeonKim/assets/90364711/a8928f29-dc68-469d-99b9-f4b331bfb3cf)